guides/v3.2.0/components/triggering-changes-with-actions.md

```diff
@@ -186,7 +186,7 @@ export default Component.extend({
 });
 ```
 
-`this.get('onConfirm')` will return the function passed from the parent as the
+`this.onConfirm` will return the function passed from the parent as the
 value of `onConfirm`, and the following `()` will invoke the function.
```

Comment:
I wonder if we still need the explanation about "the following ()". When "get()" was needed I _did_ feel like I needed the explanation of the extra parens, but now it's just invoking a function property as usual.

Reply:
yep, agree, can tweak or remove that phrase.